doc: Add quick start (#299)
TheThingsIndustries/lorawan-stack-docs: doc/content/getting-started/quick-start/index.md

weight: -3
---

This section contains everything you need to know to get you up and running on {{% tts %}}.

<!--more-->

## What is {{% tts %}}

First off, it's a good idea to understand what {{% tts %}} is. Learn that, and lots more acronyms in the [What is {{% tts %}} section]({{< ref "getting-started/what-is-tts" >}}).

## LoRaWAN

In case you skipped the previous section, here's a quick summary: {{% tts %}} is a network server which implements the LoRaWAN protocol. To learn more about LoRaWAN, check out the [Learn section](https://thethingsnetwork.org/docs) of The Things Network, or read up at the [LoRa Alliance](https://lora-alliance.org)

Here's a video of Johan Stokking, the tech lead of The Things Industries and one of the core developers of {{% tts %}}, explaining LoRaWAN:

{{< youtube "ZsVhYiX4_6o" >}}

## Start Using {{% tts %}}

To get started with {{% tts %}}, you can simply [create an account](https://console.cloud.thethings.network) on The Things Network to start using The Things Stack Community. 

If you're more of the DIY type, you can also [install and run your own network server]({{< ref "getting-started/installation" >}}).

## Build LoRaWAN Solutions

> Once you have access to {{% tts %}}, it's time to start connecting devices. See the [Adding Devices section]({{< ref "devices/adding-devices" >}}) for instructions, or watch the video below for a complete introduction.
> 
> If you're using The Things Stack Community in an urban area, you may already have gateway coverage thanks to other members of the public The Things Network community. See [The Things Network Coverage Map](https://www.thethingsnetwork.org/map) to find out.
> 
> If you're not covered, or you want to add gateways to extend the coverage of your network, see the [Adding Gateways section]({{< ref "gateways/adding-gateways" >}}).
> 
> Once you've added a device and it's connected to a gateway and to {{% tts %}}, see the [Integrations section]({{< ref "integrations" >}}) to learn how to connect other services to act on device data.

Reviewer: ```suggestion
Once you have access to {{% tts %}}, it is time to start connecting devices. See the [Adding Devices section]({{< ref "devices/adding-devices" >}}) for instructions, or watch the video below for a complete introduction.

If you are using The Things Stack Community in an urban area, you may already have gateway coverage thanks to other members of the public The Things Network community. See [The Things Network Coverage Map](https://www.thethingsnetwork.org/map) to find out.

If you are not covered, or you want to add gateways to extend the coverage of your network, see the [Adding Gateways section]({{< ref "gateways/adding-gateways" >}}).

Once you have added a device and it is connected to a gateway and to {{% tts %}}, see the [Integrations section]({{< ref "integrations" >}}) to learn how to connect other services to act on device data.
```

{{< youtube "rK8oJHZ9Q7U" >}}

## Scale Up

> When you're ready to start running your LoRaWAN solution commercially, get in touch with us at [The Things Industries](https://thethingsindustries.com) for an affordable, scalable, deployment which is SLA backed.
> 

Reviewer: ```suggestion
When you are ready to start running your LoRaWAN solution commercially, get in touch with us at [The Things Industries](https://thethingsindustries.com) for an affordable, scalable, deployment which is SLA backed.

```

## Useful Links

- Learn more about LoRaWAN and our global, crowdsourced LoRaWAN network at [The Things Network Learn](https://thethingsnetwork.org/docs)
- Learn to [add gateways]({{< ref "gateways/adding-gateways" >}}), [add devices]({{< ref "devices/adding-devices" >}}), and [create integrations]({{< ref "integrations" >}})
- Join The Things Network [Forum](https://www.thethingsnetwork.org/forum/) or [Slack](https://thethingsnetwork.slack.com/) for community support.

> - Get in touch with [The Things Industries](https://thethingsindustries.com) when you're ready to scale up

Reviewer: ```suggestion
- Get in touch with [The Things Industries](https://thethingsindustries.com) when you are ready to scale up
```
